library/Vanilla/UploadedFile.php

```diff
@@ -246,11 +256,15 @@ private function tryApplyImageProcessing(): void {
         ];
 
         if (\Gdn::config("ImageUpload.Limits.Enabled")) {
```

Comment:
Can this call to config be removed in a similar way to what you did elsewhere?

Reply:
I was discussing this with @initvector , his reasoning behind not removing this statement is because we are assuming all usages of UploadedFile are coming from the MediaModel, where that might not always be the case.

Reply:
What usages are there currently?